go/teams/chain.go

```diff
@@ -1972,6 +1981,9 @@ func (t *teamSigchainPlayer) sanityCheckInvites(mctx libkb.MetaContext,
 		if byName[key] {
 			return nil, nil, NewInviteError(fmt.Sprintf("Invite %s appears twice in invite set", key))
 		}
+		if res.MaxUses != nil && *res.MaxUses == 0 {
```

Comment:
Maybe you wanna make a constant for -1 and disallow anything smaller.

Reply:
thanks i forgot about the `-1` support. i'll add that.